dbt/models/default/default.vw_pin_status.sql

```diff
@@ -0,0 +1,160 @@
+                    -- it's mixed use.
+                WHEN
+                    (SUBSTR(oby.class, 1, 1) = '2' OR oby.class = 'OA2')
+                    AND com.class IS NOT NULL
+                    THEN TRUE
+                ELSE FALSE
+            END
+        ) AS is_mixed_use
+    FROM {{ source('iasworld', 'pardat') }} AS par
+    LEFT JOIN {{ source('iasworld', 'comdat') }} AS com
+        ON par.parid = com.parid
+        AND par.taxyr = com.taxyr
+        AND com.cur = 'Y'
+        AND com.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'dweldat') }} AS dwel
+        ON par.parid = dwel.parid
+        AND par.taxyr = dwel.taxyr
+        AND dwel.cur = 'Y'
+        AND dwel.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'oby') }} AS oby
+        ON par.parid = oby.parid
+        AND par.taxyr = oby.taxyr
+        AND oby.cur = 'Y'
+        AND oby.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'land') }} AS land
+        ON par.parid = land.parid
+        AND par.taxyr = land.taxyr
+        AND land.cur = 'Y'
+        AND land.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'aprval') }} AS aprval
+        ON par.parid = aprval.parid
+        AND par.taxyr = aprval.taxyr
+        AND aprval.cur = 'Y'
+        AND aprval.deactivat IS NULL
+    LEFT JOIN {{ source('iasworld', 'exadmn') }} AS admn
+        ON par.parid = admn.parid
+        AND par.taxyr = admn.taxyr
+        AND admn.cur = 'Y'
+        AND admn.deactivat IS NULL
+    WHERE par.cur = 'Y'
+        AND par.deactivat IS NULL
+    GROUP BY par.parid, par.taxyr
+),
+
+-- These CTEs make it easier to work with CDUs from oby, comdat, dweldat since
+-- those tables aren't unique by parid and taxyr.
+dwel_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'dweldat'),
+    cdu_column = 'cdu'
+    ) }}),
+
+com_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'comdat'),
+    cdu_column = 'user16'
+    ) }}),
+
+oby_cdu AS ({{ aggregate_cdu(
+    source_model = source('iasworld', 'oby'),
+    cdu_column = 'user16'
+    ) }})
+
+SELECT
+    pdat.parid AS pin,
+    pdat.class,
+    pdat.taxyr AS year,
+    colo.is_corner_lot,
+    ahsap.is_ahsap,
+    COALESCE(vpe.pin IS NOT NULL, FALSE) AS is_exempt,
+    COALESCE(pin.tax_bill_total = 0, NULL) AS is_zero_bill,
+    vpcc.is_parking_space,
+    vpcc.parking_space_flag_reason,
+    vpcc.is_common_area,
+    SUBSTR(pdat.parid, 11, 1) = '8' AS is_leasehold,
+    ahsap.is_mixed_use,
+    pdat.class = 'RR' AS is_railroad,
+    ptst.test_type IS NOT NULL AS is_weird,
+    ptst.test_type AS weird_reason,
+    oby.cdu AS oby_cdu,
+    cdat.cdu AS com_cdu,
+    ddat.cdu AS dwel_cdu,
+    pdat.note2 AS note,
+    pdat.class = '999' AS filler_class,
+    pdat.parid LIKE '%999%' AS filler_pin
+FROM {{ source('iasworld', 'pardat') }} AS pdat
+LEFT JOIN {{ source('spatial', 'corner') }} AS colo
+    ON SUBSTR(pdat.parid, 1, 10) = colo.pin10
+    AND pdat.taxyr = colo.year
```

Comment:
**issue (blocking):** I'm seeing a huge amount of null `is_corner_lot` values in 2023.

Reply:
Is this unexpected? I see a count of about 177,000 true pin10s per year in `spatial.parcel` and and 310,000 true pin14s per year in `default.vw_pin_status`.